deadline_cloud_extension/DeadlineCloud.pyp

```diff
@@ -1,6 +1,29 @@
+import functools
+import glob
 import os
 import sys
 
+
+def glob_add_path(base, pattern):
+    matches = list(glob.iglob(os.path.join(base, pattern)))
+    if matches:
+        return os.path.join(base, matches[0])
+    return base
+
+if "win" in sys.platform:
+    paths = [
+        "resource",
+        "modules",
+        "python",
+        "libs",
+        "*win64*",
+        "dlls"
+    ]
+    dll_path = functools.reduce(glob_add_path, paths, os.path.dirname(sys.executable))
+    # Required due to a longstanding bug in C4D to not include the python dll library path required by many compiled libs
+    #   Read more here: https://github.com/danbradham/wheels/issues/4#issuecomment-1772721170
+    os.add_dll_directory(dll_path)
```

Comment:
wondering if we should also version check this as this should not be required in newer versions. Not aware if there's negative effects in newer versions

Reply:
Shouldn't cause issue but I can  version gate it below 2024.2 if you'd prefer

Reply:
Would definitely be my preference (so that it's easier to remove workarounds) but we can add that later. We at least have the paper trail.